docs/source/hub.rst

```diff
@@ -4,6 +4,227 @@
 Hub Module
 ==========
 
-To be written.
+.. contents::
+    :depth: 3
 
-If you'd like to author this topic, please assign yourself `MOTECH-964 <https://applab.atlassian.net/browse/MOTECH-964>`_.
+############
+Introduction
+############
+
+The Hub module is an implementation of PubSubHubbub Hub specification (PubSubHubbub is a simple publication - subscription
+protocol). MOTECH Hub implementation exposes API for publishers and subscribers, allowing publishing and distributing
+content to all interested subscribers when subscribed resources are updated. With help of MOTECH Hub any external module
+can subscribe to a specific resource and be notified when it change as well as can act as publisher to make content
```

Comment:
This sentence is a small mess. Perhaps move the part starting with "as well as" to a new sentence.